README.md

```diff
@@ -78,6 +78,16 @@ widgets:
     - 't3news'
 ```
 
+### MFA Providers
+
+```
+widgets: ['*']
```

Comment:
I guess this should be called `mfa-providers` ?

Reply:
@bmack yes of course  - changed now!